.github/workflows/test.yml

```diff
@@ -44,3 +47,58 @@ jobs:
           source activate          
           pip install pytest
           pytest
+
+  paths-filter:
```

Comment:
Can we make this a step in the update-model-request-template job instead of making it a separate job?

Reply:
done